pkg/plugins/plugins.go

```diff
@@ -73,6 +73,12 @@ type Plugin struct {
 	activateWait *sync.Cond
 }
 
+// PathsRelativeTo returns the path to which all paths returned by the plugin are relative to.
+// For legacy plugins, this is always relative to the host's root directory.
+func (p *Plugin) PathsRelativeTo() string {
```

Comment:
@anusha-ragunathan the problem is that for legacy plugin the notion of "pluginroot" is kind of moot. This is returning the "basepath" to which all paths returned by the plugin will be relative to.